### ISSUE TITLE
Updating sim credits 

### ISSUE THREAD

**Issue by emily-phet** [phetsims/faradays-law#189]
We need to make sure:
- Taliesin
- Jesse
- Ashton
- Mike Winters
- Emily 
- John Blanco
- Jon Hung (re magnet alternative input design)
- Michael Barlow

are included, anyone else?

**Comment by terracoda** [comment]
We need to add Caren Watkins to the list.

**Comment by terracoda** [comment]
Jon Hung and Caren Watkins both worked on the first draft of interactive description for this sim, so I would want to include them both as collaborators from the IDRC.

**Comment by terracoda** [comment]
The way we did this in John Travoltage was in addition to the Team list we added another list called "Collaborators" and the collaborators and their institution were added on the sim credits page, like this:

- Collaborators: Jonathan Hung, Caren Watkins (Inclusive Design Research Centre)

I suggest we do it the same way for Faraday's Law. While Jonathan Hung's alternative input design for the magnet is wonderfully innovative, it is not the only thing he contributed for Faraday's Law.

**Comment by terracoda** [comment]
![Screen Shot 2020-10-28 at 11 30 44 AM](https://user-images.githubusercontent.com/1902116/97446165-0b521800-1911-11eb-932f-a0959839cde4.png)

@jbphet, I see you got everyone there already except Caren Watkins.

Just add Caren, and change Emily's name to Emily B. Moore and I think we have the Team and collaborators complete.
 The QA may have additions afterward testing.

**Comment by jessegreenberg** [comment]
I did very little work in this sim other than general support, I am going to remove my name from credits. Otherwise list seems good to me!

**Comment by jbphet** [comment]
I added Caren Watkins.  @terracoda told me that she too works for the IDRC, and it was a little verbose to have "Jonathan Hung (Interactive Design Research Center), Caren Watkins (Interactive Design Research Center)", so I removed the "(Interactive Design Research Center)" part and just listed the two names.  Is that cool?  Alternatively, we could say "Jonathan Hung & Caren Watkins (Interactive Design Research Center)" if it's important to have the IDRC's full name in there.

**Comment by terracoda** [comment]
Yes that's great, but the institution is the "Inclusive Design Research Centre" and in Canada we spell it, "Centre" with the e and the r reversed, so that is not a typo.

**Comment by jbphet** [comment]
> Yes that's great...

Which option did you thing was great?

**Comment by jbphet** [comment]
> Let's go with "Jonathan Hung & Caren Watkins (Interactive Design Research Centre)", nothing, as @terracoda did, that the correct spelling is "Centre" not "Center".

And that it's "Inclusive", not "Interactive".

**Comment by jbphet** [comment]
Okay, I've added the IDRC part as specified.  I'm going to unassign this and leave it open, since there will almost certainly be more names to add when QA is done.

**Comment by terracoda** [comment]
Apologies @jbphet, I thought everything was clear (in https://github.com/phetsims/faradays-law/issues/189#issuecomment-717926005) given in John Travoltage there was no repeat of the institution name.
- "Jonathan Hung & Caren Watkins (Inclusive Design Research Centre)"

The dialog looks good now, and you say, you'll likely need to add some QA folks.

<img width="646" alt="Screen Shot 2020-11-04 at 10 57 47 AM" src="https://user-images.githubusercontent.com/1902116/98123598-a3f61400-1e8c-11eb-9462-97ed9e93139c.png">

**Comment by jbphet** [comment]
@KatieWoe - I think we're getting close to publication, so please let me know who else should be added to the QA credits.

**Comment by KatieWoe** [comment]
Jaspe Arias, Devon Quispe, and Brooklyn Lash should be added so far

**Comment by jbphet** [comment]
Thanks @KatieWoe - those three names have been added to both master and the 1.4 branch.

**Comment by jbphet** [comment]
Just to be clear, this issue pertains to what will be the 1.4 release.

**Comment by phet-steele** [comment]
This looks great in 1.4.0-rc.2!
